File: app/src/main/java/cn/nicolite/archer/jBase/BasePresenter.java
package cn.nicolite.archer.jBase;


import cn.nicolite.mvp.jBase.JBasePresenter;

/**
 * Presenter基类  所有Persenter都要继承此类
 * Created by nicolite on 17-10-13.
 */

public abstract class BasePresenter<I, V> extends JBasePresenter<I, V> {

    public BasePresenter(I iView, V view) {
        super(iView, view);
    }
}

File: app/src/main/java/cn/nicolite/archer/kBase/BasePresenter.kt
package cn.nicolite.archer.kBase

import cn.nicolite.mvp.kBase.KBasePresenter

/**
 * Created by nicolite on 2018/5/20.
 * email nicolite@nicolite.cn
 */
abstract class BasePresenter<I, V>(iView: I, view: V) : KBasePresenter<I, V>(iView, view) {

}